site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,235 @@
++++
+date = "2018-05-26T11:30:00+05:30"
+title = "Citra Progress Report 2018 Q1"
+tags = [ "progress-report" ]
+author = "CaptV0rt3x"
+forum = 0000
++++
+
+It's been a while folks. Since the last report, we have had many new features come to Citra. Some
```

Comment:
`It's been a while folks - Since the last ...`
(otherwise, this is a sentence fragment)